### PR TITLE
ISSUE_TEMPLATE & PR_TEMPLATE added.

### LINKED ISSUE #28: Add Issue template and PR template
A unified workflow for how New Issue can be created and PR are supposed to be made lacks right now.

### PR DESCRIPTION
**Pull request by Purvanshsingh**
Fixes #28 

Templates are inside the docs folder.

### CODE REVIEW

**Comment by Purvanshsingh** [comment]
@tripathyprateek Please review the Pull Request.

**Comment by tripathyprateek** [comment]
@Purvanshsingh Issue template and Pull Request template need to be inside of a new folder (.github) as ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE.md. I am attaching links for how its done and requesting you to follow that. Let me know if you have any queries regarding this.

https://github.blog/2016-02-17-issue-and-pull-request-templates/
https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates

**Review by tripathyprateek — CHANGES_REQUESTED** [review]
Please make the necessary changes and follow the links I have mentioned.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Edit these points below to describe the new features added with this PR -->
+<!-- - Feature 1 -->
+<!-- - Feature 2 -->
+
+
+#### Changed
+<!-- Edit these points below to describe the changes made in existing functionality with this PR -->
+<!-- - Change 1 -->
+<!-- - Change 1 -->
+
```
Comment (tripathyprateek) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: add a line for providing Screenshots here.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+### Change logs
+
+<!-- #### Added -->
+<!-- Edit these points below to describe the new features added with this PR -->
+<!-- - Feature 1 -->
+<!-- - Feature 2 -->
+
+
+#### Changed
```
Comment (tripathyprateek) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: this is not needed, you have written change logs above

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+<!-- - Change 1 -->
+<!-- - Change 1 -->
+
+
+#### Fixed
+<!-- Edit these points below to describe the bug fixes made with this PR -->
+<!-- - Bug 1 -->
+
+
+#### Removed
```
Comment (tripathyprateek) [inline .github/PULL_REQUEST_TEMPLATE.md:31]: this is also not needed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Purvanshsingh) [reply .github/PULL_REQUEST_TEMPLATE.md:20]: which one should I remove changed or change logs?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (Purvanshsingh) [reply .github/PULL_REQUEST_TEMPLATE.md]: Resolved.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Purvanshsingh) [reply .github/PULL_REQUEST_TEMPLATE.md]: resolved.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Purvanshsingh) [reply .github/PULL_REQUEST_TEMPLATE.md]: Resolved.

**Comment by Purvanshsingh** [comment]
@tripathyprateek made the required changes.

**Review by tripathyprateek — APPROVED** [review]
Awesome job. Thanks a lot @Purvanshsingh . Approving and merging 🚀

**Comment by Purvanshsingh** [comment]
@tripathyprateek what are the grading guidelines for PR? , how this PR will be counted for SWOC?

**Pull request merged**